save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/FileUploader.kt

```diff
@@ -80,7 +97,7 @@ fun fileUploader(
     onFileSelect: (HTMLSelectElement) -> Unit,
     onFileRemove: (FileInfo) -> Unit,
     onFileInput: (HTMLInputElement) -> Unit,
-    onExecutableChange: (file: FileInfo, checked: Boolean) -> Unit,
+    onExecutableChange: (file: FileInfo, checked: Boolean) -> Unit
```

Comment:
Why? Trailing comma is useful here (btw, we should configure diktat rule for it already)

Reply:
ok, fixed